pkg/utils/mark.go

```diff
@@ -0,0 +1,36 @@
+package utils
```

Comment:
can we please add some docs on this package

Reply:
yes